playbooks/roles/foreman_installer/tasks/install.yml

```diff
@@ -0,0 +1,9 @@
+---
+- name: 'Run installer'
+  shell: >
+      foreman-installer {{ (foreman_installer_verbose == True) | ternary("-v", "") }}
+      --scenario "{{ foreman_installer_scenario }}"
+      --foreman-admin-password "{{ foreman_installer_admin_password }}"
+      {{ foreman_installer_options }}
+      {{ foreman_installer_options_internal_use_only | join(" ") }}
```

Comment:
I think this looks great, so take tis question as a looking ahead. This seems ripe for an Ansible module. Would you agree or do you think that would be overkill? e.g.

```
- name: 'Run Installer'
  foreman_installer:
    scenario: foreman
    verbose: false
    options: >
      -- foreman-admin-password "changeme"
```